### DISCUSSION TITLE
Reaction to OOM kick?

### DISCUSSION THREAD

**Discussion by snowleopardd** [YimMenu/YimMenu#3374] [Q&A] [answered]
i can't find it

**Comment by snowleopardd** [comment, marked as answer]
closing this as i found out there is no reaction for it.

**Comment by ghost** [comment]
> i can't find it

As far as I remember, when an oom kick is triggered, the person is identified as a modder and we can kick/block him with a reaction

**Reply by snowleopardd** [reply]
Yeah indeed but if you just want to co-exist with other friendly modders (not friends btw) you just can't

Also the cheater detection is kinda inaccurate sometimes, i would not rely on it for reactions

**Reply by ghost** [reply]
> Yeah indeed but if you just want to co-exist with other friendly modders (not friends btw) you just can't
> 
> Also the cheater detection is kinda inaccurate sometimes, i would not rely on it for reactions

where is the inaccuracy?  false positive crashes do not detect the player as a modder.  or removal of the vehicle.  There are no false positives in the modder's detection.  or at least I didn't notice.

if you want to coexist with modders in a session, then check the "trust this session" box

**Reply by lonelybud** [reply]
> Also the cheater detection is kinda inaccurate sometimes, i would not rely on it for reactions

some reactions can be false positives.

**Reply by ghost** [reply]
> > Also the cheater detection is kinda inaccurate sometimes, i would not rely on it for reactions
> 
> some reactions can be false positives.

yes, this applies to crashes, deleting vehicles, spectate on me and maybe something else. but with the modder detection I didn’t notice any false positive detections
